### ISSUE TITLE
Validation dataset

### ISSUE THREAD

**Issue by cooperlab** [CancerDataScience/HistomicsML#7]
@slee172 The datasets used for training and validation are typically going to be different. We should add a drop-down under "Validate classifier" so that the dataset can be chosen separately for the classifier and the validation set.

Suggested names:
"Dataset" -> "Training dataset"
"Training set" -> "Classifier name"
*Add a new "Validation dataset" drop-down
"Test set" -> "Validation set name"

**Comment by slee172** [comment]
Added a drop down list for Training dataset and modified names.
